### PR TITLE
Add update task for Cargo

### PR DESCRIPTION
**Pull request by rbyshko**

### CODE REVIEW

**Comment by rbyshko** [comment]
Added dependency on `cargoBuildSupport`. Next I'll add update task for Poetry, in this PR.

File: src/kraken/std/cargo/__init__.py [hunk 1]
```diff
@@ -146,6 +148,11 @@ def cargo_fmt(*, all_packages: bool = False, project: Project | None = None) ->
     project.do("cargoFmtCheck", CargoFmtTask, all_packages=all_packages, group="lint", check=True)
 
 
+def cargo_update(*, project: Project | None = None) -> CargoUpdateTask:
+    project = project or Project.current()
+    return project.do("cargoUpdate", CargoUpdateTask, group="update")
```
Comment (NiklasRosenstein) [inline src/kraken/std/cargo/__init__.py:153]: > Added dependency on `cargoBuildSupport`. Next I'll add update task for Poetry, in this PR.

Where is it? I was expecting to find this here 👀

Comment (rbyshko) [reply src/kraken/std/cargo/__init__.py]: I actually added it to the our kraken, now moved here, see the commit below.

**Comment by rbyshko** [comment]
@NiklasRosenstein this is ready to merge - please review.

**Pull request merged**